client/Makefile

```diff
@@ -1,8 +1,10 @@
 BACKEND_VMM ?= $(shell pkg-config --cflags vchan | sed 's/.*vchan-//')
+ROOTDIR := $(patsubst %client/,%,$(dir $(abspath $(lastword $(MAKEFILE_LIST)))))
 
 CFLAGS += -I../include -g -Wall -Wextra -Werror -pie
 CFLAGS += -fPIC
 CFLAGS += -DBACKEND_VMM_$(BACKEND_VMM)
+CFLAGS += -ffile-prefix-map=$(ROOTDIR)=.
```

Comment:
I think this should be set by some rpm macro (similar like dpkg-buildpackage does), otherwise we'll need to add this to every package we build, but also possibly this could conflict with the Debian settings at some point.

Maybe we need `qubes-rpm-config` package similar as there is `redhat-rpm-config`?

Reply:
I think this is a good idea. I need to figure some cases like Makefile.am in gui-agent for which I'm having some issues while building with rpmbuild but not with direct build.